### DISCUSSION TITLE
Is it possible to use a fixed SHA for ubuntu-latest?

### DISCUSSION THREAD

**Discussion by carlosribas** [actions/virtual-environments#7730] [General questions] [answered]
I'm getting an error on my docker container that was built with GitHub Actions and I'm trying to test if it is related to the latest version of ubuntu-latest. I tried to use a fixed SHA by using `ubuntu-latest@2150a217b0252a095e386ffa176efd243c6eec8b`, but the workflow is stuck at Queue state for a long time. Does anyone know if it's possible to use a fixed SHA? Thanks

**Comment by vpolikarpov-akvelon** [comment, marked as answer]
Hi, @carlosribas. It's not possible to choose specific version of runner image. Moreover GitHub serves only the latest version of each image type (unless an update rollout is in progress) so it wouldn't work anyway.

**Reply by carlosribas** [reply]
Hi @vpolikarpov-akvelon. Okay, thank you.
